cmd/svcat/parameters/parameters_test.go

```diff
@@ -77,6 +77,40 @@ func TestParseVariableAssignments_OneVariableThreeValues(t *testing.T) {
 		t.Fatalf("%s\nexpected:\n\t%v\ngot:\n\t%v\n", "one var three values", want, got)
 	}
 }
+func TestParseVariableAssignments_InvalidDotParams(t *testing.T) {
+	params := []string{"a.=e"}
```

Comment:
This would benefit from using subtests, see below for an example. Then you can quickly add variations of this test for `.a=e` or `a.b..=e` just to cover all the edge cases.

https://github.com/kubernetes-incubator/service-catalog/blob/c51705f264350f54244b337ac8c87d71503c8731/cmd/svcat/svcat_test.go#L62-L76

Reply:
Perfect, will do that - thank you!